deb-arm/Dockerfile

```diff
@@ -112,6 +123,23 @@ RUN if [ "$DD_TARGET_ARCH" = "armhf" ] ; then \
 ENV PATH="/usr/local/go/bin:${PATH}"
 ENV PATH="${GOPATH}/bin:${PATH}"
 
+# Rust is needed to compile the SDS library
+RUN if [ "$DD_TARGET_ARCH" = "armhf" ] ; then \
+           export RUSTUP_SHA256=${RUSTUP_SHA256_ARMHF} \
+        && export RUSTUP_ARCH=${RUSTUP_ARCH_ARMHF} \
+        && export RUSTC_SHA256=${RUSTC_SHA256_ARMHF} \
```

Comment:
small nit: you could simplify by doing `export FOO=bar BAR=baz ...` in a single command, which should reduce the boilerplate a bit